code/modules/economy/cash.dm

```diff
@@ -17,50 +17,34 @@
 	var/worth = 0
 
 /obj/item/weapon/spacecash/c1
-	name = "1 credip chip"
```

Comment:
Yeah, them being named manually was redundant with New() doing it for you (and replacing the names). I removed it from the individual definitions so there wasn't the impression that those assigned names were actually being used anywhere.